unittests/InstructionCountCI/AVX128/VEX_map1.json

```diff
@@ -0,0 +1,6318 @@
+        "movk w20, #0x1, lsl #16",
+        "str x20, [x28, #272]",
+        "mov w1, #0x401",
+        "str x1, [x28, #1328]",
+        "ldr x0, [x28, #2296]",
+        "br x0"
+      ]
+    },
+    "vmovss [rax], xmm0": {
+      "ExpectedInstructionCount": 7,
+      "Comment": [
+        "Map 1 0b10 0x11 128-bit"
+      ],
+      "ExpectedArm64ASM": [
+        "mov w20, #0xac4",
+        "movk w20, #0x1, lsl #16",
+        "str x20, [x28, #272]",
+        "mov w1, #0x401",
+        "str x1, [x28, #1328]",
+        "ldr x0, [x28, #2296]",
+        "br x0"
+      ]
+    },
+    "db 0xc5, 0xf2, 0x11, 0xc2": {
+      "ExpectedInstructionCount": 7,
+      "Comment": [
+        "vmovss xmm2, xmm1, xmm0",
+        "Need to manually encode since nasm won't encode this",
+        "Map 1 0b10 0x11 128-bit"
+      ],
+      "ExpectedArm64ASM": [
+        "mov w20, #0xb64",
+        "movk w20, #0x1, lsl #16",
+        "str x20, [x28, #272]",
+        "mov w1, #0x401",
+        "str x1, [x28, #1328]",
+        "ldr x0, [x28, #2296]",
+        "br x0"
+      ]
+    },
+    "vmovsd [rax], xmm0": {
+      "ExpectedInstructionCount": 7,
+      "Comment": [
+        "Map 1 0b11 0x11 128-bit"
+      ],
+      "ExpectedArm64ASM": [
+        "mov w20, #0xc04",
+        "movk w20, #0x1, lsl #16",
+        "str x20, [x28, #272]",
+        "mov w1, #0x401",
+        "str x1, [x28, #1328]",
+        "ldr x0, [x28, #2296]",
+        "br x0"
+      ]
+    },
+    "db 0xc5, 0xf3, 0x11, 0xc2": {
+      "ExpectedInstructionCount": 7,
+      "Comment": [
+        "vmovsd xmm2, xmm1, xmm0",
+        "Need to manually encode since nasm won't encode this",
+        "Insert in to first element could be more optimal, which is the common case.",
+        "Map 1 0b11 0x11 128-bit"
+      ],
+      "ExpectedArm64ASM": [
+        "mov w20, #0xca4",
+        "movk w20, #0x1, lsl #16",
+        "str x20, [x28, #272]",
+        "mov w1, #0x401",
+        "str x1, [x28, #1328]",
+        "ldr x0, [x28, #2296]",
+        "br x0"
+      ]
+    },
+    "vmovlps xmm0, xmm1, [rax]": {
+      "ExpectedInstructionCount": 7,
+      "Comment": [
+        "Insert in to first element could be more optimal, which is the common case.",
+        "Map 1 0b00 0x12 128-bit"
+      ],
+      "ExpectedArm64ASM": [
+        "mov w20, #0xd44",
+        "movk w20, #0x1, lsl #16",
+        "str x20, [x28, #272]",
+        "mov w1, #0x401",
+        "str x1, [x28, #1328]",
+        "ldr x0, [x28, #2296]",
+        "br x0"
+      ]
+    },
+    "vmovlpd xmm0, xmm1, [rax]": {
+      "ExpectedInstructionCount": 7,
+      "Comment": [
+        "Insert in to first element could be more optimal, which is the common case.",
+        "Map 1 0b01 0x12 128-bit"
+      ],
+      "ExpectedArm64ASM": [
+        "mov w20, #0xde4",
+        "movk w20, #0x1, lsl #16",
+        "str x20, [x28, #272]",
+        "mov w1, #0x401",
+        "str x1, [x28, #1328]",
+        "ldr x0, [x28, #2296]",
+        "br x0"
```

Comment:
I think something is very wrong here.

Reply:
I think the instcountci commit needs to be interchanged with the "enable AVX" commit.

Reply:
Oops, instcountci was disabling AVX if SVE256 was explicitly disabled. Weird interaction that no longer needs to exist.